ceph/hack/test-pod.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test
+spec:
+  containers:
+    - name: curl-runner
+      image: docker.io/yuvalif/test-ceph-notifications
```

Comment:
I'd recommend not doing this - let's use `ko` specific path, that other in kn land do too